### DISCUSSION TITLE
Approving a PR is confusing

### DISCUSSION THREAD

**Discussion by fabioferreira3** [community/community#11441] [Mobile]
Everytime I need to review a PR I spend lota of minutes trying to find the right button to redirect me to the review page. 
It's not clear that you need to find the 'updated files' button first, THEN you'll see the review changes button.
The review page should have its own action within the main PR page.

**Comment by brianlovin** [comment]
Thanks for the feedback @fabioferreira3 — in many ways, we're trying to match the user experience of the website where you have to first navigate to the Files Changed subtab on a PR before you're able to leave a review. There may be room for us to improve the entry point though in the PR header so that it's more clear what the next required step is for a viewer.
